src/simulation/simulator.cpp

```diff
@@ -69,7 +69,7 @@ Simulator::Simulator(MainWindow *mainWindow) : QObject(mainWindow) {
 
 	QSettings settings;
 	int enabled = settings.value("simulatorEnabled", 0).toInt();
-	enable(enabled);
+	enable(true);
```

Comment:
I think it is now save to remove that setting entirely.

While on it, will it make sense to have a switch between DC and transient analysis?

Reply:
I am OK with removing the enable simulator setting.